src/stats.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * This software is licensed under the terms of the MIT License.
+ * See COPYING for further information.
+ * ---
+ * Copyright (c) 2011-2019, Lukas Weber <laochailan@web.de>.
+ * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
+ */
+
+#ifndef IGUARD_stats_h
+#define IGUARD_stats_h
+
+#include "taisei.h"
+
+typedef struct Stats Stats;
+
+struct Stats {
+	struct {
+		int lives;
```

Comment:
I think this is a little misleading, because it can be interpreted as either "lives used" or "total lives gained". I would rename it either `deaths` or `lives_used`; perhaps the same applies to bombs.